runtime/lua/vim/treesitter/_fold.lua

```diff
@@ -69,7 +74,10 @@ local function compute_folds_levels(bufnr, info, srow, erow, parse_injections)
   srow = srow or 0
   erow = erow or api.nvim_buf_line_count(bufnr)
 
-  local parser = assert(ts.get_parser(bufnr, nil, { error = false }))
+  local parser = info.parser
+  if not parser then
```

Comment:
Right now the `assert` makes the user aware that `get_parser` returned `nil`. Is it okay to change this to a silent return?